manifests/packages.pp

```diff
@@ -13,22 +13,18 @@
 #   package prefix `$php::package_prefix`
 #
 class php::packages (
-  $ensure          = $::php::ensure,
-  $manage_repos    = $::php::manage_repos,
-  $names_to_prefix = prefix(
+  String $ensure         = $::php::ensure,
+  Boolean $manage_repos  = $::php::manage_repos,
+  Array $names_to_prefix = prefix(
     $::php::params::common_package_suffixes, $::php::package_prefix # lint:ignore:parameter_documentation
```

Comment:
We should be consistent about how we merge two values for a parameter name. Compare to `manifests/embedded.pp` L18 or so.